src/css/support.js

```diff
@@ -0,0 +1,53 @@
+import jQuery from "../core.js";
+import document from "../var/document.js";
+import documentElement from "../var/documentElement.js";
+import support from "../var/support.js";
+
+( function() {
+
+var reliableTrDimensionsVal,
+	div = document.createElement( "div" );
+
+// Finish early in limited (non-browser) environments
+if ( !div.style ) {
+	return;
```

Comment:
Reaching here will make `support.reliableTrDimensions()` crash. I guess this is similar to what we do on `3.x-stable`...

Reply:
I thought about that. An error is probably a good thing in that case, but perhaps there's a better place for the error?

Reply:
It might be OK to leave it here.